openedx/adg/lms/webinars/admin.py

```diff
@@ -8,15 +8,8 @@
 
 from openedx.adg.lms.applications.admin import adg_admin_site
 
-from .constants import (
-    WEBINAR_REGISTRATION_DELETE_PERMISSION_GROUP,
-    WEBINAR_STATUS_CANCELLED,
-    WEBINAR_STATUS_DELIVERED,
-    WEBINAR_STATUS_DRAFT,
-    WEBINAR_STATUS_UPCOMING
-)
-from .forms import WebinarForm
-from .models import CancelledWebinar, Webinar, WebinarRegistration
+from .constants import WEBINAR_STATUS_CANCELLED, WEBINAR_STATUS_DELIVERED, WEBINAR_STATUS_DRAFT, WEBINAR_STATUS_UPCOMING
+from .models import Webinar, WebinarRegistration
 
 
 class ActiveWebinarStatusFilter(admin.SimpleListFilter):
```

Comment:
`status` field from webinars is removed. I don't know who removed and why but choices for that field are still in webinar model. Due to the status removal, Webinar list page is breaking in django admin, Pls update.